app/Http/Controllers/AuthController.php

```diff
@@ -64,17 +64,19 @@ public function login(LoginRequest $request)
             if (!$token) {
                 return new BaseResponse(STATUS_CODE_BADREQUEST, STATUS_CODE_BADREQUEST, "Incorrect email or password");
             }
-    
+
             if ((!Auth::guard('api')->user()->is_verified)) {
```

Comment:
This one as well, If this gets done from Middleware before accessing the controller that would be good.